app/test_main.py

```diff
@@ -1,3 +1,16 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize("cat_age, dog_age, expected", [
+    (0, 0, [0, 0]),
+    (14, 14, [0, 0]),
+    (15, 15, [1, 1]),
+    (23, 23, [1, 1]),
+    (24, 24, [2, 2]),
+    (27, 27, [2, 2]),
+    (28, 28, [3, 2]),
+    (100, 100, [21, 17])
+])
+def test_get_human_age(cat_age: int, dog_age: int, expected: int) -> None:
+    assert get_human_age(cat_age, dog_age) == expected
```

Comment:
How do you handle this
![Screenshot from 2023-09-19 14-01-54](https://github.com/mate-academy/py-cat-and-dog-years/assets/122517203/4e543787-b540-46a3-b1e1-66b15695ed57)